tests/api.c

```diff
@@ -64569,6 +64569,73 @@ static int test_dtls_downgrade_scr(void)
 }
 #endif
 
+/* Make sure we don't send acks before getting a server hello */
```

Comment:
Need to update this comment to reflect that we are testing the changecipherspec message being dropped. @JacobBarthelmeh if you have a moment you can do this and push to my branch to make the merge today.

Reply:
Thanks @JacobBarthelmeh!